src/main/java/com/sparta/popupstore/s3/S3ImageService.java

```diff
@@ -39,6 +41,20 @@ public S3UrlResponseDto getPreSignedUrl(String prefix, String fileName){
                 .build();
     }
 
+    public List<S3UrlResponseDto> getPreSignedUrls(String prefix, List<ImageRequestDto> imageRequestDto) {
+        return imageRequestDto
+                .stream()
+                .map(image -> {
+                 this.validFileName(image.getFileName());
+                 String createFileName = String.format("%s/%s", prefix, this.createUuid() + image.getFileName());
+                 String preSignedUrl = generateResignedUrlRequest(createFileName);
+                 return S3UrlResponseDto.builder()
+                         .preSignedUrl(preSignedUrl)
+                         .imageUrl(baseUrl +createFileName)
+                         .build();
```

Comment:
```suggestion
                 return getPreSignedUrl(prefix, image.getFileName());
```
getPreSignedUrl 메서드 하나로 줄일 수 있을 것 같습니다.

Reply:
아 그러네요!!!!! 헉